app/User.php

```diff
@@ -66,4 +66,15 @@ public function setPassWordAttribute($passcode)
     {
         $this->attributes['password'] = Hash::make($passcode);
     }
+
+    /**
+     * @return \Illuminate\Database\Eloquent\Relations\HasMany
+     * one to many: user can submit many survey responses
+     */
+
+    public function surveyResponse()
+    {
+        return $this->hasMany(SurveyResponse::class);
+    }
```

Comment:
Same thing here, are we referring to the users taking the survey or those participating in it?
If the latter then I think this may not be useful since most surveys don't collect information that identifies the respondents

Reply:
The former. The user (enumerator) conducting the survey. The respondent does not interact with the system. The enumerator asks the questions and fills out the responses provided by the respondent, and then submits the response. We are not collecting any information that identifies the respondent. But we would like to know who administered the survey (the user/enumerator)